.github/workflows/run_tests.yml

```
@@ -0,0 +1,25 @@
+name: Run Maven Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
```

I think we should keep the JDK version consistent for all github actions 
```suggestion
      - name: Set up JDK 1.8
```